applications/crossbar/src/modules/cb_dialplans.erl

```diff
@@ -63,11 +64,23 @@ resource_exists() -> 'true'.
 %%--------------------------------------------------------------------
 -spec validate(cb_context:context()) -> cb_context:context().
 validate(Context) ->
-    Doc = maybe_add_name(kapps_config:get_all_kvs(<<"dialplans">>)),
+    Doc = maybe_add_name(get_joined_dialplans(cb_context:account_id(Context))),
     cb_context:setters(Context, [{fun cb_context:set_resp_data/2, Doc}
                                 ,{fun cb_context:set_resp_status/2, 'success'}
                                 ]).
 
+-spec get_joined_dialplans(api_binary()) -> [kz_json:object()].
+get_joined_dialplans('undefined') -> get_dialplans('undefined');
+get_joined_dialplans(AccountId) ->
+    get_dialplans(AccountId) ++ get_dialplans('undefined').
+
+-spec get_dialplans(api_binary()) -> [kz_json:object()].
+get_dialplans('undefined') ->
+    kapps_config:get_all_kvs(<<"dialplans">>);
+get_dialplans(AccountId) ->
+    JObj = kapps_account_config:get_global(AccountId, ?ACCOUNT_CONFIG, <<"dialplans">>, {[]}),
```

Comment:
system_schemas/CONFIG gets transalated to accountdb/configs_CONFIG document.
please review kapps_account_config and see if all paths are valid in getting/setting the value.
thanks